css/order.css

```diff
@@ -69,29 +67,43 @@
   flex: 2;
 }
 
-.bnomics-altcoin-pane {
-  align-items: center;
-  flex: 1;
-  text-align: center;
+.bnomics-grey {
+  color:grey;
 }
 
 .bnomics-address {
+  margin-top: 10px;
   position:relative;
   margin:auto !important;
-  width: 90% !important;
 }
 
-.bnomics-address input , .bnomics-address select {
-  font-weight: 600 !important;
-  margin-top: 5px !important;
-  margin-bottom: 10px !important;
-  width: 90% !important;
-  border: 1px solid #DDD; !important;
-  background-color: #fff !important;
-  box-shadow: none !important;
+.bnomics-amount{
+  margin-top: 10px;
```

Comment:
One small ui change, the top margin can be removed from bnomics-amount as it is shifting the alignment of the text to the qrcode.

Reply:
Removed the top margin from bnomics-amount.